Simplified/Book/UI/NYPLBookCellDelegate+AudiobookBookmark.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  NYPLBookCellDelegate+AudiobookBookmark.swift
+//  Simplified
+//
+//  Created by Ernest Fan on 2022-10-21.
+//  Copyright © 2022 NYPL. All rights reserved.
+//
+
+#if FEATURE_AUDIOBOOKS
+import Foundation
+import NYPLAudiobookToolkit
+import UIKit
+
+/// The NYPLAudiobookBookmarksBusinessLogic is not accessible from ObjC,
+/// so we do this in a Swift extension file.
+@objc extension NYPLBookCellDelegate {
+  @objc(setBookmarkBusinessLogicForBook:AudiobookManager:)
+  func setBookmarkBusinessLogic(for book: NYPLBook, _ audiobookManager: DefaultAudiobookManager) {
```

Comment:
can you inject at least the BookRegistry, since it's already present where this is called from?